app/javascript/menu/first-level.jsx

```diff
@@ -84,6 +86,7 @@ MenuSection.propTypes = {
   items: PropTypes.arrayOf(PropTypes.any).isRequired,
   onSelect: PropTypes.func.isRequired,
   title: PropTypes.string.isRequired,
+  idx: PropTypes.number.isRequired,
```

Comment:
Minor, but I'm not a fan of abbreviation at the API level. Can we spell this out to index, or perhaps use another work like position?